api/README.md

```diff
@@ -4,21 +4,17 @@
 
 In the project directory, you can run:
 
-### `npm install`
+### `yarn install`
 
-`npm install` installs all dependencies to make this project work.
+`yarn install` installs all dependencies to make this project work.
 
-### `npm run setup --plugins`
+### `yarn run seed`
```

Comment:
Here we can just use `yarn seed` - we can remove `run` from the command.
This comment is applicable in other commands.

Reply:
Done.  I adjusted the remaining commands.